pkg/csi/service/wcpguest/controller.go

```diff
@@ -307,6 +326,121 @@ func (c *controller) ControllerPublishVolume(ctx context.Context, req *csi.Contr
 	}
 	log.Debugf("Found virtualMachine instance for node: %q", req.NodeId)
 
+	// File volumes support
+	if isFileVolumeRequest {
```

Comment:
Can we refactor the code to have separate method that handles `ControllerPublishVolume` for block volume and another one for file volume? This helps in code readability.

Reply:
Done